### PR TITLE
fix: updates to wallet asset schemas

### PR DESCRIPTION
**Pull request by cbachmeier**
NftBalance schema had a minor alternation when moved to prod, this supports that alteration. Listed profile Cards should load as expected

### CODE REVIEW

File: src/graphql/data/nft/NftBalance.ts [hunk 1]
```diff
@@ -148,10 +148,10 @@ export function useNftBalanceQuery(
       lastPrice: queryAsset.node.lastPrice?.value,
       floorPrice: asset.collection?.markets[0]?.floorPrice?.value,
       creatorPercentage: queryAsset.node.listingFees && queryAsset.node.listingFees[0].basisPoints / 10000,
-      listing_date: asset.listings ? asset.listings[0]?.edges[0]?.node.createdAt : undefined,
+      listing_date: asset.listings?.edges[0] ? asset.listings?.edges[0]?.node?.createdAt : undefined,
       date_acquired: queryAsset.node.lastPrice?.timestamp,
-      sellOrders: asset.listings?.edges,
-      floor_sell_order_price: asset.listings ? asset.listings[0]?.edges[0]?.node.price.value : undefined,
+      sellOrders: asset.listings?.edges.map((edge: any) => edge.node),
```
Comment (vm) [inline src/graphql/data/nft/NftBalance.ts:153]: why the any type here?

File: src/nft/components/profile/view/WalletAssetDisplay.tsx [hunk 2]
```diff
@@ -100,7 +100,7 @@ export const WalletAssetDisplay = ({ asset, isSellMode }: { asset: WalletAsset;
                           borderRadius="4"
                           marginLeft="4"
                           objectFit="cover"
-                          src={`/nft/svgs/marketplaces/${market.toLowerCase()}.svg`}
+                          src={`/nft/svgs/marketplaces/${market?.toLowerCase()}.svg`}
```
Comment (vm) [inline src/nft/components/profile/view/WalletAssetDisplay.tsx:103]: if the market is unavailable should we not have a src at all?

Comment (cbachmeier) [reply src/nft/components/profile/view/WalletAssetDisplay.tsx]: good call

File: src/graphql/data/nft/NftBalance.ts (hunk 1 above)
Comment (cbachmeier) [reply src/graphql/data/nft/NftBalance.ts]: This was a convenience, I've updated the casted type to use the NonNullable syntax.

**Review by vm — APPROVED** [review]

File: src/graphql/data/nft/NftBalance.ts [hunk 3]
```diff
@@ -127,31 +134,31 @@ export function useNftBalanceQuery(
...
+      name: asset?.name,
+      tokenId: asset?.tokenId,
       asset_contract: {
-        address: asset.collection?.nftContracts[0].address,
-        schema_name: asset.collection?.nftContracts[0].standard,
-        name: asset.collection?.name,
-        description: asset.description,
-        image_url: asset.collection?.image?.url,
-        payout_address: queryAsset.node.listingFees && queryAsset.node.listingFees[0].payoutAddress,
+        address: asset?.collection?.nftContracts && asset?.collection?.nftContracts[0]?.address,
```
[the remark this answers is not in the document]
Comment (cbachmeier) [reply src/graphql/data/nft/NftBalance.ts]: I regret not knowing this syntax until now

**Pull request merged**